inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/render/RecommendationSpanRenderer.java

```diff
@@ -188,29 +184,26 @@ public void render(CAS aCas, VDocument vdoc, AnnotatorState aState,
                 
                 boolean first = true;
                 Map<Long, AnnotationSuggestion> confidencePerClassifier = labelMap.get(label);
-                for (Long recommenderId: confidencePerClassifier.keySet()) {
```

Comment:
I don't understand the changes that follow here... what's the purpose?

Reply:
In order to reduce cognitive intelligibility, I want to split a large nested loop into two small loops, but the comprehensibility of the code is very unfavorable.